### DISCUSSION TITLE
PG_AUTO_FAILOVER IN DMZ

### DISCUSSION THREAD

**Discussion by dbaplessis** [hapostgres/pg_auto_failover#1050] [Q&A]
I implemented the pg_auto_failover solution in Production. On applications that are in DMZ, there are unwanted switches. Is there a specific configuration of PG_AUTO_FAILOVER for firewalls? How to do it ?

**Comment by dbaplessis** [comment]
Forgive me, I forgot the greetings.
Hello everyone. I repeat m’y question, I need Your Yelp, please :  
I implemented the pg_auto_failover solution in Production. On applications that are in DMZ, there are unwanted switches. Is there a specific configuration of PG_AUTO_FAILOVER for firewalls ? How to do it ?

**Reply by krishnakongara123** [reply]
same issue, i'm going through!
switches are happening on it's own randomly! 
and logs are not indicating clear reason about the issue!

@dimitri  - appreciate if you can shed some light here!
why /bin/journalctl -u pg_auto_failover.service, is not giving proper logging!
is there any parameters, we need to set to track that culprit ?
